resources/stage_4_repair/MTRT - MS Telemetry Removal/MTRT.cmd

```diff
@@ -0,0 +1,316 @@
+		CD /D "%SYSTEMDRIVE%\ProgramData\Microsoft\Diagnosis\ETLLogs\AutoLogger"
+		CALL :LOGCMD TAKEOWN /F AutoLogger-Diagtrack-Listener.etl
+		CALL :LOGCMD ICACLS AutoLogger-Diagtrack-Listener.etl /GRANT ADMINISTRATORS:F /Q
+		CALL :LOGCMD BREAK>AutoLogger-Diagtrack-Listener.etl
+		CALL :LOGCMD ICACLS AutoLogger-Diagtrack-Listener.etl /DENY SYSTEM:F /Q
+	) ELSE (
+		CALL :LOGCMD MKDIR "%SYSTEMDRIVE%\ProgramData\Microsoft\Diagnosis\ETLLogs\AutoLogger"
+		CALL :LOGCMD CD /D "%SYSTEMDRIVE%\ProgramData\Microsoft\Diagnosis\ETLLogs\AutoLogger"
+		CALL :LOGCMD COPY /Y NUL AutoLogger-Diagtrack-Listener.etl
+		CALL :LOGCMD ICACLS AutoLogger-Diagtrack-Listener.etl /DENY SYSTEM:F /Q
+)
+CALL :LOGCMD REG ADD "HKLM\SYSTEM\CurrentControlSet\Services\SharedAccess\Parameters\FirewallPolicy\FirewallRules" /v "{60E6D465-398E-4850-BE86-7EF7620A2377}" /t REG_SZ /d "v2.24|Action=Block|Active=TRUE|Dir=Out|App=C:\windows\system32\svchost.exe|Svc=DiagTrack|Name=Windows Telemetry|" /f
+CD /D "%DATA%"
+
+REM Start Window 10 Only
+IF /I NOT "%WIN_VER:~0,9%"=="Windows 1" (GOTO:SKIP_10_TWEAKS)
+REM Disabled for Tron; it seems regular Flash also stores files here
+REM CALL :LOGTXT "   Kill Adobe Flash that ships with Windows 10"
+REM RD /S /Q "%APPDATA%\Adobe" >NUL 2>&1
+REM RD /S /Q "%APPDATA%\Macromedia" >NUL 2>&1
+REM TAKEOWN /F "%WINDIR%\System32\Macromed" /A /R /D Y >NUL 2>&1
+REM ICACLS "%WINDIR%\System32\Macromed" /GRANT ADMINISTRATORS:F /T /Q >NUL 2>&1
+REM RD /S /Q "%WINDIR%\System32\Macromed" >NUL 2>&1
+REM TAKEOWN /F "%WINDIR%\SysWOW64\Macromed" /A /R /D Y >NUL 2>&1
+REM ICACLS "%WINDIR%\SysWOW64\Macromed" /GRANT ADMINISTRATORS:F /T /Q >NUL 2>&1
+REM RD /S /Q "%WINDIR%\SysWOW64\Macromed" >NUL 2>&1
+
+CALL :LOGTXT "   Kill OneDrive integration"
+CALL :LOGCMD TASKKILL /F /IM  Explorer.exe /T
+CALL :LOGCMD %SystemRoot%\System32\OneDriveSetup.exe /Uninstall
+CALL :LOGCMD %SystemRoot%\SysWOW64\OneDriveSetup.exe /Uninstall
+REM These keys are orphaned after the OneDrive uninstallation and can be safely removed
+CALL :LOGCMD REG DELETE "HKCR\CLSID\{018D5C66-4533-4307-9B53-224DE2ED1FE6}" /F
+CALL :LOGCMD REG DELETE "HKCR%Wow6432Node%\CLSID\{018D5C66-4533-4307-9B53-224DE2ED1FE6}" /F
+CD /D "%USERPROFILE%\AppData\Roaming\Microsoft\Windows\Start Menu\Programs"
+CALL :LOGCMD DEL "OneDrive.lnk" /F /S /Q
+FOR %%D IN ("%LOCALAPPDATA%\Microsoft\OneDrive","%USERPROFILE%\OneDrive","%PROGRAMDATA%\Microsoft OneDrive","%SYSTEMDRIVE%\OneDriveTemp") DO (
+	CALL :LOGCMD TAKEOWN /F %%D /A /R /D Y
+	CALL :LOGCMD ICACLS %%D /GRANT ADMINISTRATORS:F /T /Q
+	CALL :LOGCMD DEL %%D /F /S /Q
+	CALL :LOGCMD RMDIR %%D /S /Q
+)
+CD /D "%WINDIR%\WinSxS"
+FOR /D %%O IN (*onedrive*) DO (
+	CALL :LOGCMD TAKEOWN /F %%O /A /R /D Y
+	CALL :LOGCMD ICACLS %%O /GRANT ADMINISTRATORS:F /T /Q
+	CALL :LOGCMD DEL %%O /F /S /Q
+	CALL :LOGCMD RMDIR %%O /S /Q
+)
+CALL :LOGCMD START Explorer.exe
+CD /D "%DATA%"
+
+CALL :LOGTXT "   Disable geolocation Service"
```

Comment:
Do not want to disable this - this has legitimate uses, especially on mobile devices.